src/rpc/net.cpp

```diff
@@ -272,6 +272,49 @@ static RPCHelpMan getpeerinfo()
     };
 }
 
+static RPCHelpMan addwhitelist()
+{
+    return RPCHelpMan{"addwhitelist",
```

Comment:
Wouldn't it be better to call this RPC `addpermissionflags`, to better convey that this is setting flags and not a single black/white bool?

Reply:
+1 -- we also have an rpc whitelist, so making it clear this changes net permission flags, not the rpc whitelist itself seems good